mast/testRunner.mt

```diff
@@ -163,6 +163,37 @@ def makeAsserter() as DeepFrozen:
                     catch _:
                         successes += 1
 
+                to contains(o, k):
+                    if (o._respondsTo("contains", 1)):
```

Comment:
It allows for error message customization. Pretty much all of these methods are about customized errors and easier-to-read testing flows.